src/request/import/Import.css

```diff
@@ -70,3 +70,14 @@ button.go-to-words {
     margin: 4rem;
     margin-right: 0;
 }
+
+.page#choose-key-type button:not([disabled]),
+.page#words button:not([disabled]) {
+    background: #724ceb;
+    color: white;
+}
+
+.page#choose-key-type button,
+.page#words button {
+    margin: 0 7.5rem 4rem;
+}
```

Comment:
Try to configure your editor to automatically append a final line break at the end of a file.